specification/cost-management/resource-manager/Microsoft.CostManagement/stable/2021-10-01/costmanagement.json

```diff
@@ -1178,7 +1178,7 @@
           "ReservedInstances"
         ],
         "operationId": "GenerateReservationDetailsReport_ByBillingProfileId",
-        "description": "Generates the reservations details report for provided date range asynchronously by billing profile.",
+        "description": "Generates the reservations details report for provided date range asynchronously by billing profile. The Reservation usage details can be viewed by only certain enterprise roles by default. For more details on the roles see, https://docs.microsoft.com/en-us/azure/cost-management-billing/reservations/reservation-utilization#view-utilization-in-the-azure-portal-with-azure-rbac-access",
```

Comment:
nit:  reword " viewed by only certain enterprise roles by default" to " viewed only by certain enterprise roles"